main.tf

```
@@ -68,7 +68,7 @@ data "aws_iam_policy_document" "policy" {
       }
 
       dynamic "condition" {
-        for_each = try(statement.value.conditions, [])
+        for_each = try(statement.value.condition, [])
```

we can add both to ensure we don't break for existing users like:

```suggestion
        for_each = try(statement.value.conditions, statement.value.condition, [])
```

---

I have added this suggestion to main.tf, thank you